layers/meta-resin-beaglebone/recipes-kernel/linux/linux-beagleboard_4.4.bb

```diff
@@ -21,9 +21,9 @@ COMPATIBLE_MACHINE = "beaglebone"
 
 S = "${WORKDIR}/git"
 
-TAG = "4.4.21-ti-rt-r46"
+TAG = "4.4.55-ti-rt-r94"
```

Comment:
Yes, the kernel bump is needed to get the Beaglebone Blue dts included in the kernel.